build_prod.py

```diff
@@ -51,6 +51,50 @@ def bash_run_command(cmd):
     if ret != 0:
         raise Exception('Failed to run ' + cmd + ': ' + str(ret))
 
+# block to work with pull requests
+
+def verify_pulls(existed, proposed, uri):
+    print('Verifying of the proposed pulls: {}'.format(proposed))
+    for pr in proposed:
+         print('Verifying {}'.format(pr))
+         if not pr in existed:
+             print('Pull request {} does not exist in {} repository.'.format(pr, uri))
+             return False
+    return True
+
+
+def get_pulls_for(uri):
+    print('Getting the pulls for {}'.format(uri))
+    g = Github()
+    repo = g.get_repo(uri)
+    pulls = repo.get_pulls(state='open', sort='created', base='master')
+    res = []
+    for pr in pulls:
+        res.append(pr.number)
+    print(res)
```

Comment:
Is this for debug only or how does it look like printed? Could you please give a sample output?

Reply:
> Is this for debug only or how does it look like printed? Could you please give a sample output?

fixed